react/components/Portal.tsx

```diff
@@ -2,12 +2,16 @@ import React from 'react'
 import { FunctionComponent } from 'react'
 import ReactDOM from 'react-dom'
 
+import { useCssHandles } from 'vtex.css-handles'
+
 export interface Props {
   cover?: boolean
   target?: HTMLElement
   zIndex?: number
 }
 
+const CSS_HANDLES = ['autoCompleteContainer'] as const
```

Comment:
This app is used on others apps as well, so naming this handle `autoCompleteContainer` would cause a lot of confusion. What do you think about `portalContainer` or something like that?